app/code/Magento/Shipping/Model/CarrierFactoryInterface.php

```diff
@@ -38,15 +38,15 @@ public function create($carrierCode, $storeId = null);
      * @return bool|AbstractCarrierInterface
      * @api
      */
-    public function getIfActive($carrierCode);
+    public function getIfActive(string $carrierCode, int $storeId, bool $isReturn);
 
     /**
      * Create carrier by its code if it is active
      *
      * @param string $carrierCode
-     * @param null|int $storeId
+     * @param int $storeId
      * @return bool|AbstractCarrierInterface
      * @api
      */
-    public function createIfActive($carrierCode, $storeId = null);
+    public function createIfActive(string $carrierCode, int $storeId, bool $isReturn);
```

Comment:
New methods or method parameters cannot be added into existing `@api` interfaces as it will break every existing interface implementation. Please check http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#introduction-of-a-method-to-a-class-or-interface